src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -2,20 +2,370 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using System.Globalization;
 using Microsoft.Xunit.Performance;
+using Xunit;
 
 namespace System.Tests
 {
     public class Perf_Double
     {
         public static readonly double dValue = 1.23456789E+5;
+        private const int iterCount = 20000;
 
-        [Benchmark]
-        public static void Double_ToString()
+        [Benchmark(InnerIterationCount = iterCount)]
+        [InlineData(104234.343)]
+        [InlineData(double.MinValue / 2)]
+        [InlineData(Math.PI)]
+        [InlineData(Math.E)]
+        [InlineData(double.MaxValue)]
+        [InlineData(double.MinValue)]
+        [InlineData(double.NaN)]
+        [InlineData(double.PositiveInfinity)]
+        [InlineData(double.NegativeInfinity)]
+        [InlineData(0xFFFFFFFFFFFFF)]
```

Comment:
Oops. Forgot it is CSharp. Cannot do something like (*(double*)&t). I'll change these hex values.